### DISCUSSION TITLE
Jumper T-Pro V2 TPROV2-PRODUCTION-10-5-23.zip as firmware

### DISCUSSION THREAD

**Discussion by buemonet** [EdgeTX/edgetx#4209] [General]
Hallo, after install TPROV2-PRODUCTION-10-5-23.zip on my Jumper T-Pro V2 the display was right. 

If I start EdgeTX on the Jumper T-Pro V2 all seems okay, but if I open the Channel Monitor and I change the sticks for CH1, CH2, CH3 and CH4 nothing changed on the display between +100 and -100.

The next problem ist that my Deltang receiver RX43D-1 V510 did not bind with the procedure in Setup.

What went wrong and what you have as solution?

Thank you for help.

**Comment by pfeerick** [comment]
Have you done a stick calibration yet? SYS -> HARDWARE -> Calibration

As far as the binding, I have no idea as I don't know what protocol the "Deltang receiver RX43D-1 V510" speaks, or if you have the correct version of the TProV2... 4IN1 MPM?

**Reply by drcpedro** [reply]
SYS => HARDWARE => Calibration), I can't find that option in the menu Could you help me

**Reply by pfeerick** [reply]
Sometimes a video is better ;) Long press on the first button for SYS. Since you've got a TPro V2, also center and move the two pots at the top when instructed to center and move the sticks. 

https://github.com/EdgeTX/edgetx/assets/5500713/5680f960-70e7-4cb1-a357-ec0b1cc73117

**Reply by drcpedro** [reply]
Muchas gracias amigo, ya he conseguido que todo corra bien.
